docs/client/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "CLI tools"
+weight: 06
+type: "docs"
+showlandingtoc: "false"
+aliases:
+  - /docs/reference/resources
+---
+
+The following CLI tools are available for use with Knative, either for installation, resource creation and modification, or development purposes, as explained in the description for each tool.
+
+## kubectl CLI
+
+You can use the `kubectl` CLI to apply the YAML files required to install Knative components, and also to create Knative resources, such as services and event sources using YAML.
+
+### Next steps
+
+- <a href="https://kubernetes.io/docs/tasks/tools/install-kubectl/" target="_blank">Install the `kubectl` CLI</a>
+
+## kn CLI
+
+The `kn` CLI cannot be used to install Knative components, but provides a quick and easy interface for creating Knative resources such as services and event sources, without the need to create or modify YAML files directly. The `kn` CLI also simplifies completion of otherwise complex procedures such as autoscaling and traffic splitting.
+
+### Next steps
+
+- [Install the `kn` CLI](./install-kn/)
+- [Connect the `kn` CLI to your cluster](./install-kn#connecting-the-kn-cli-to-your-cluster)
```

Comment:
`kn` and `kubectl` share the same configuration files to connect to the cluster, so either add this to `kubectl`, too, or leave it out here.

Reply:
Good to know, I'll maybe move that whole section to here in the `_index` file then